css/style.css

```diff
@@ -134,6 +134,7 @@ input {
 } */
 
 progress {
+  /* todo: there should be no whitespace between the unit and the value */
   height: 20 px;
```

Comment:
Can be read about here: [just one example doc](https://developer.mozilla.org/en-US/docs/Web/CSS/length)

![2019-06-27_01-17-19](https://user-images.githubusercontent.com/30473157/60241593-51a8ea00-9879-11e9-88ee-55a003ed9982.png)